lib/kitchen/provisioner/chef/common_sandbox.rb

```diff
@@ -86,6 +88,14 @@ def all_files_in_cookbooks
             select { |fn| File.file?(fn) && ! %w[. ..].include?(fn) }
         end
 
+        # @return [String] an absolute path to a Policyfile, relative to the
+        #   kitchen root
+        # @api private
+        def policyfile
```

Comment:
This logic is also duplicated for the berks and librarian resolvers. If you want I could refactor all of them, but I didn't want to have one pre-refactored and the others not-refactored.